.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Seems like these .vscode files should not be included in the repo.   Perhaps add them to your .gitignore

Reply:
Makes sense, will do

Reply:
Nvm, I thought it didn't work when I did moved it to the project folder originally but now seems to work.  the top-level git-ignore is now back to normal.